include/LLVMSPIRVOpts.h

```diff
@@ -0,0 +1,104 @@
+//===- LLVMSPIRVOpts.h - Specify options for translation --------*- C++ -*-===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
```

Comment:
As this is a new file, would it make sense to update the license header / copyright line to the new ones? (see #80 and #134).

Reply:
I'm not sure about this particular one: I'm not very familiar with all this licensing stuff, but this file is going to be shipped together with the translator for integration with other libraries. That effectively means that downstream users that use pre-built packages will get a package with two different licenses - is it okay?

Reply:
No idea to be honest; perhaps easier to leave it as is for now then.